src/routes/(console)/organization-[organization]/settings/deleteOrganizationModal.svelte

```diff
@@ -55,7 +55,7 @@
             showDelete = false;
             addNotification({
                 type: 'success',
-                message: `${$organization.name} has been flagged for deletion`
+                message: `${$organization.name} has been deleted`
```

Comment:
The linear task says that the copy needs to change only when a free org is deleted, the old copy still works for pro and scale orgs